### PR TITLE
Done

### PR DESCRIPTION
**Pull request by mel-d**
This passes all of the tests, but I still need to look around a bit more to figure out how these files fit together.

One question: I made a new file for each page within the website (in the views folder). Was that the correct thing to do?

### CODE REVIEW

**Comment by mel-d** [comment]
^Also, it's telling me that a Travis CI build failed--does that mean there's something wrong with my code or is that unrelated?

**Comment by sumeetjain** [comment]
> does that mean there's something wrong with my code or is that unrelated?

You have capitalized `Greet` and also lowercased `greet` for the same files. That needs to be addressed.

> I made a new file for each page within the website (in the views folder). Was that the correct thing to do?

Yup!

**Comment by mel-d** [comment]
I just found where you mean--I hadn't realized that capitalization would be so important. Fixed it!

**Comment by mel-d** [comment]
When I tried to push again, though, it told me "Everything's up to date". Are you able to see the change on your end?

**Comment by halfghaninne** [comment]
Did you commit before pushing?

File: controllers/main_controller.rb [hunk 1]
```diff
@@ -9,3 +9,19 @@
...
+
+MyApp.get "/about" do
+  erb :"main/about"
+end
+
+MyApp.get "/favorites" do
+  erb :"main/favorites"
+end
+
+MyApp.get "/greet/Sumeet" do
```
Comment (sumeetjain) [inline controllers/main_controller.rb]: What happens if you delete this action entirely?

Comment (mel-d) [inline controllers/main_controller.rb]: When I deleted those lines, I got four errors all with this message:   1) Failure:
MoreTest#test_links_on_favorites [/Users/melindadeinert/Code/web-toy__simple-application/tests/more_test.rb:40]:
Failed assertion, no message given.

**Comment by mel-d** [comment]
Alex, I did commit! But I actually got a message that the commit was 'clean' or something; that nothing had been changed.

File: controllers/main_controller.rb (hunk 1 above)
Comment (sumeetjain) [inline controllers/main_controller.rb]: Really? Can you make the changes you made and add/commit/push? I'd like to see.

Comment (mel-d) [inline controllers/main_controller.rb]: Yes, just for the record I didn't actually delete them, but I commented them out. I'll re-push right now.

**Comment by sumeetjain** [comment]
Gotcha, I see the miscommunication. I only wanted you to try to delete the action for `"/greet/Sumeet"`. Give that a try and see how your code changes and what tests fail.

**Comment by mel-d** [comment]
It definitely makes sense that I wouldn't need that extra /greet action. My tests are all still passing, but this Travis CI build thing is still failing.

**Comment by sumeetjain** [comment]
Yeah, that capitalization issue will keep it failing. No worries though. You're good to move on–just be consistent with capitalization moving forward :)

**Pull request closed without merging**